main_windows.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"errors"
+	"github.com/GoogleCloudPlatform/cloud-sql-proxy/v2/cmd"
```

Comment:
Let's move all the imports for the proxy and third-party libs below the standard library imports with a blank line separating them.

cf. https://groups.google.com/g/golang-dev/c/fvWvubpTEds

Reply:
np, I literally have no opinion about it. Usually leave it to Goland to handle the imports 🤷